skins/moono-lexicon/icons/icons.json

```diff
@@ -1,141 +1,39 @@
-            "output": "justifyright"
-        },
-        {
-            "source": "align-justify",
-            "output": "justifyblock"
-        },
-        {
-            "source": "link",
-            "output": "link"
-        },
-        {
-            "source": "chain-broken",
-            "output": "unlink"
-        },
-        {
-            "source": "flag-full",
-            "output": "anchor"
-        },
-        {
-            "source": "picture",
-            "output": "image"
-        },
-        {
-            "source": "table",
-            "output": "table"
-        },
-        {
-            "source": "separator",
-            "output": "horizontalrule"
-        },
-        {
-            "source": "full-size",
-            "output": "maximize"
-        },
-        {
-            "source": "change-list",
-            "output": "replace"
-        },
-        {
-            "source": "search",
-            "output": "find"
-        },
-        {
-            "source": "undo",
-            "output": "undo"
-        },
-        {
-            "source": "redo",
-            "output": "redo"
-        },
-        {
-            "source": "paste",
-            "output": "copy"
-        },
-        {
-            "source": "cut",
-            "output": "cut"
-        },
-        {
-            "source": "code",
-            "output": "source"
-        },
-        {
-            "source": "hidden",
-            "output": "hidden"
-        },
-        {
-            "source": "align-image-center",
-            "output": "align-image-center"
-        },
-        {
-            "source": "align-image-left",
-            "output": "align-image-left"
-        },
-        {
-            "source": "align-image-right",
-            "output": "align-image-right"
-        }
-    ]
+    "icons": {
+        "bold": "bold",
+        "italic": "italic",
+        "underline": "underline",
+        "strikethrough": "strike",
+        "subscript": "subscript",
+        "superscript": "superscript",
+        "remove-style": "removeformat",
+        "list-ol": "numberedlist",
+        "list-ul": "bulletedlist",
+        "indent-more": "indent",
+        "indent-less": "outdent",
+        "quote-right": "blockquote",
+        "align-left": "justifyleft",
+        "align-center": "justifycenter",
+        "align-right": "justifyright",
+        "align-justify": "justifyblock",
+        "link": "link",
+        "chain-broken": "unlink",
+        "flag-full": "anchor",
+        "picture": "image",
+        "table2": "table",
+        "separator": "horizontalrule",
+        "full-size": "maximize",
+        "change-list": "replace",
+        "search": "find",
+        "undo": "undo",
+        "redo": "redo",
+        "paste": "copy",
+        "cut": "cut",
+        "code": "source",
+        "hidden": "hidden",
+        "align-image-center": "align-image-center",
+        "align-image-left": "align-image-left",
+        "align-image-right": "align-image-right"
```

Comment:
Actually I thought about inverting key-value so the key is CKEditor button name and value the Clay icon name, makes more sense in a skin (and I'm thinking now that it could solve possible issues if we need to use the same Clay icon in more than one button)... So... you'll get your sorting...

![image](https://media.giphy.com/media/l2JdU7e38RqzdlakU/giphy.gif)